R/makeQset.R

```diff
@@ -24,8 +24,8 @@
 #' @return A qseaSet object, containing all the information required.
 #' @export
 makeQset <- function(sampleTable,
-                     BSgenome = NULL,
-                     chrSelect = 1:22,
+                     BSgenome,
+                     chrSelect,
```

Comment:
The function documentation should be updated as this has no longer a default. Also, does the code handle the fact that by mistake the user may not supply chromosomes?

If not, either keep the default, or make an explicit error in case `chrSelect` is null, I'd say.

Reply:
As it has no default, the function won't work if you don't provide anything, it'll give an error like `argument "chrSelect" is missing, with no default`